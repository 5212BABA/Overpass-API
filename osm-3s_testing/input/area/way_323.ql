way(323);node(area)(if:id()>=30600 && id()<=30699);out;
