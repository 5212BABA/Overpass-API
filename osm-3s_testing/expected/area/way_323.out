<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <node id="30600" lat="51.2458880" lon="7.1499776"/>
  <node id="30601" lat="51.2458879" lon="7.1499776"/>
  <node id="30602" lat="51.2458879" lon="7.1499777"/>
  <node id="30603" lat="51.2458880" lon="7.1499777"/>
  <node id="30604" lat="51.2458881" lon="7.1499777"/>
  <node id="30605" lat="51.2458881" lon="7.1499776"/>
  <node id="30606" lat="51.2458881" lon="7.1499775"/>
  <node id="30607" lat="51.2458880" lon="7.1499775"/>
  <node id="30608" lat="51.2458879" lon="7.1499775"/>
  <node id="30620" lat="51.2458880" lon="7.1499808"/>
  <node id="30621" lat="51.2458879" lon="7.1499808"/>
  <node id="30622" lat="51.2458879" lon="7.1499809"/>
  <node id="30623" lat="51.2458880" lon="7.1499809"/>
  <node id="30624" lat="51.2458881" lon="7.1499809"/>
  <node id="30625" lat="51.2458881" lon="7.1499808"/>
  <node id="30626" lat="51.2458881" lon="7.1499807"/>
  <node id="30627" lat="51.2458880" lon="7.1499807"/>
  <node id="30628" lat="51.2458879" lon="7.1499807"/>
  <node id="30640" lat="51.2458912" lon="7.1499776"/>
  <node id="30641" lat="51.2458911" lon="7.1499776"/>
  <node id="30642" lat="51.2458911" lon="7.1499777"/>
  <node id="30643" lat="51.2458912" lon="7.1499777"/>
  <node id="30644" lat="51.2458913" lon="7.1499777"/>
  <node id="30645" lat="51.2458913" lon="7.1499776"/>
  <node id="30646" lat="51.2458913" lon="7.1499775"/>
  <node id="30647" lat="51.2458912" lon="7.1499775"/>
  <node id="30648" lat="51.2458911" lon="7.1499775"/>

</osm>
